casepro/backend/junebug.py

```diff
@@ -40,6 +46,60 @@ def get_addresses(self, uuid):
         return (
             a['address'] for a in addresses if a.get('address') is not None)
 
+    def get_identities(self, **kwargs):
+        '''Get the list of identities filtered by the given kwargs.'''
+        url = '%s/api/v1/identities/search/?' % (self.base_url)
+        for key, value in kwargs.iteritems():
+            url = '%s%s=%s&' % (url, key, value)
+
+        identities = self.get_paginated_response(
+            url, params={'default': True})
+
+        return (
+            IdentityStoreContact(i) for i in identities if
+            i.get('details').get('name') is not "removed"
+        )
+
+
+class IdentityStoreContact(object):
+    """
+    Holds identity data for syncing
+    """
+    def __init__(self, json_data):
+        self.id = json_data.get('id')
```

Comment:
I wonder whether it would be better to just set all the values from the data as attributes? E.g.
```python
for k, v in json_data.items():
    setattr(self, k, v)
```
?